space-concordia-robotics/robotics-prototype
# [#433] Fix battery_temp ThermistorTemps values displayed on the GUI

## Description
Changes made to be able to display battery_temps values onto GUI. Previously, the values were never displayed and remained constantly as "NaN" as opposed to their published values. This fixes the bug where values were never properly displayed on the GUI. 

### Steps for Testing
1. Run rosgui and startgui in two terminal windows simultaneously.
2. Open the GUI on http://localhost:5000.
3. In terminal, publish a set of 3 battery temperatures using: `rostopic pub /battery_temps mcu_control/ThermistorTemps "therm1: <value1> therm2: <value2> therm3: <value3>"`
4. Observe the values displayed on the GUI.

closes #433
